lib/matplotlib/ticker.py

```diff
@@ -168,6 +169,15 @@
     long = int
 
 
+# Work around numpy/numpy#6127.
+def divmod(x, y):
```

Comment:
A quick grep through all other calls to divmod in the codebase suggests that they are always called either with integer arguments (in which case there's no bug) or with (nontrivial) array arguments (in which case the fix I used is not enough).  So I'll just leave it here (as `_divmod`) for now.

The ternary-if has also been fixed.